src/coding.h

```diff
@@ -756,6 +746,18 @@ extern Lisp_Object from_unicode_buffer (const wchar_t *wstr);
   } while (false)
 
 
```

Comment:
Why did this move?

Reply:
The upstream changed that in f3fafd24175a8.